### ISSUE TITLE
Contrastive loss in the model

### ISSUE THREAD

**Issue by fbao-fudan** [vinid/path_eval#9]
Dear authors, 

Thanks for releasing the code. I noticed the manuscript mentioned you used contrastive loss in the model. Could you please point me to the position where to use it? 
Thank you!

Feng

**Comment by vinid** [comment]
Hi!

Thanks for your interest!

The code in this repo cannot be used for training. What we have here is to do reproducibility studies.

A couple of starting points that might be useful to get into the contrastive loss:

* [Training Code based on OriginalCLIP](https://github.com/openai/CLIP/issues/83) (this is the one we used for training)
* [HF Training Pipeline](https://github.com/huggingface/transformers/tree/main/examples/pytorch/contrastive-image-text)
* [OpenCLIP training pipeline](https://github.com/mlfoundations/open_clip) (this is the one we recently started using for training, due to the fact that it's easier to scale to multi-GPU setups with this one)

Hope this is helpful!

**Comment by fbao-fudan** [comment]
Thank you! This is really helpful.
